fullcontact/api/verification_api.py

```diff
@@ -0,0 +1,50 @@
+from concurrent.futures import Future
+
+from .base.base_api import BaseApi
+from ..response.verification_response import EmailVerificationResponse
+from ..schema.verification_schema import EmailVerificationRequestSchema
+
+
+class VerificationApi(BaseApi):
+    _BASE_URL = "https://api.fullcontact.com/v2/"
+
+    _email_verification_endpoint = "verification/email"
+    _email_verification_request = EmailVerificationRequestSchema()
+    _email_verification_response = EmailVerificationResponse
+
+    def email(self, email: str, headers: dict = None) -> _email_verification_response:
+        r"""
+        POST the email to FullContact email verification API
```

Comment:
This and the call below is really a GET right?

Reply:
Yes, it is a GET. An error while copying. Fixed!